src/Microsoft.OData.Core/ODataBatchWriter.cs

```diff
@@ -313,81 +313,34 @@ public Task<ODataBatchOperationResponseMessage> CreateOperationResponseMessageAs
 #endif
 
         /// <summary>Flushes the write buffer to the underlying stream.</summary>
-        public void Flush()
-        {
-            this.VerifyCanFlush(true);
-

```

Comment:
Why is this made abstract? The implementation should be the same in both the JsonLight and MultipartMixed derived classes, you just have to change this.rawOutputContext.Flush() to this.outputContext.Flush(), then you can remove the overrides in the derived classes.